emsdk.py

```diff
@@ -3116,7 +3119,14 @@ def print_tools(t):
         tool = find_sdk(arg)
         if tool is None:
           error_on_missing_tool(arg)
-      tools_to_activate += [tool]
+
+      if cmd == 'activate':
+        tools_to_activate += [tool]
+      elif tool in tools_to_activate:
+        print('Deactivating tool ' + str(tool) + '.')
+        tools_to_activate.remove(tool)
```

Comment:
So "remove" is something that already exists?   Is it analogous to "deactivate" ?  What is it used for today?

Reply:
Oh sorry.. I thought this was calling a method called `remove()` .. my bad